src/space-connector/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { ResponseCode } from './type';
+
+export const RESPONSE: {[key in string]: ResponseCode} = {
+    API_ERROR: 'APIError',
+    BAD_REQUEST_ERROR: 'BadRequestError',
+    NOT_FOUND_ERROR: 'NotFoundError',
+    AUTHENTICATION_ERROR: 'AuthenticationError',
+    AUTHORIZATION_ERROR: 'AuthorizationError'
+};
```

Comment:
```suggestion
export const RESPONSE = {
    API_ERROR: 'APIError',
    BAD_REQUEST_ERROR: 'BadRequestError',
    NOT_FOUND_ERROR: 'NotFoundError',
    AUTHENTICATION_ERROR: 'AuthenticationError',
    AUTHORIZATION_ERROR: 'AuthorizationError'
} as const;
```

위와 같은 방식을 사용하지 않은 이유가 궁금합니다!